src/Slider.tsx

```diff
@@ -12,17 +11,23 @@ interface SlideVisibilityEntry {
     visibility: Visibility;
 }
 
-interface Settings {
+export interface SliderSettings {
```

Comment:
We might want to namespace this when exporting, so we don't 'pollute' the global types of people using the package